Automattic/woocommerce-payments
# feat: digital wallets onboarding page

Fixes #

#### Changes proposed in this Pull Request









Ensures the merchant can enable/disable "Digital wallets"/"1-click checkouts" from the onboarding page.

#### Testing instructions






- Ensure you have the latest version of the dev tools: `cd docker/wordpress/wp-content/plugins/woocommerce-payments-dev-tools && git checkout trunk && git pull`
- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Ensure you have the "Add UPE additional payment methods" flag enabled in WCPay Dev Tools
- Visit http://localhost:8082/wp-admin/admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods
- Enable the "Enable Apple Pay & Google Pay" checkbox
- Click "Continue"
- Refresh the page
- The settings are persisted
- On the settings page, the same payment methods/1-click checkout status should be displayed

![2021-05-25 14 47 33](https://user-images.githubusercontent.com/273592/119559367-2d592f00-bd68-11eb-9593-80c3e875bde3.gif)


-------------------
